static/css/eval_rig.scss

```diff
@@ -152,7 +153,6 @@ label {
   }
 
   .feedback-nav-section {
-    margin-top: auto;
```

Comment:
I added this to fix the position of buttons.

Reply:
yes, the flex-grow I added accomplishes the same thing, but keeps the border above the buttons at the same height, otherwise it keeps moving when we switch pages

Reply:
I see, i missed the flex-grow change.